Security/Certificate Pinning.md

```diff
@@ -0,0 +1,80 @@
+Certificate pinning can help improve the security of an app by ensuring that the app only trusts a specific set of certificates, rather than all of the certificates in the system’s trust store. This can also help prevent man-in-the-middle attacks, where an attacker attempts to intercept and modify network traffic.
+
+Android N brought several new features and improvements, one of which is the [Network Security Config](https://developer.android.com/training/articles/security-config#CertificatePinning). This feature allows developers to specify a custom XML configuration file that enables certificate pinning for their app, even when using WebView.
+
+One issue with certificate pinning is that `android:networkSecurityConfig` is not backward compatible with older versions of Android. Another issue is that it can be difficult to implement when using self-signed certificates.
+
+One solution is to use an internally forked library [TrustKit for Android](https://github.com/infinum/TrustKit-Android/tree/feature/trust-anchors).
```

Comment:
The last commit on this repo was more than 2 years ago, do we still want to link this?

Reply:
Definitely a good point. That fork is being actively used on RBA / RBA mBiz, but we should see about maintaining it and updating it from time to time with new code in the original TrustKit repository.

@bosko-nikcevic-infinum could you inform one of the LEs about this? That you wrote a new chapter about using our fork of TrustKit and that you're worried about that fork not being up to date.

Reply:
Will do if you think it's needed since the last release on original repo was in April of 2020, prior to the fork.